packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -398,7 +410,7 @@ export class RelationshipsController extends TransportController {
     }
 
     @log()
-    private async completeStateTransition(targetStatus: RelationshipStatus, id: CoreId) {
+    private async executeNonDeletingOperation(operation: RelationshipOperation, id: CoreId) {
```

Comment:
I don't like the new method name... and you should not rename `RelationshipAuditLogEntryReason` to `RelationshipOperation` and therefore just define a completely new concept that nobody knows.

Reply:
I didn't want to put an AuditLog in there - then I suggest changing the name RelationshipAuditLogEntryReason everywhere.

And completeStateTransition won't be enough once the reactivation is implemented (of course, the nonDeleting only matters when the decomposition is implemented, I could remove that)

Reply:
What's the matter with using another method for the reactivation? (or think about that for the reactivation)

Please undo all that for now that it doesn't get confusing.

Reply:
this method does the updating + event issuing - we'd end up duplicating this

Reply:
ok, let's think about this when we implement it. undo it for now.

Reply:
alright